apps/a1/Dockerfile.merged

```
@@ -0,0 +1,9 @@
+# Simulate deployed service for local testing.
+FROM us-west1-docker.pkg.dev/serverless-runtimes/google-22/runtimes/python313
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)